src/pages/settings/Wallet/WalletPage/WalletPage.tsx

```diff
@@ -467,7 +467,7 @@ function WalletPage({bankAccountList = {}, cardList = {}, fundList = {}, isLoadi
                                                         <Button
                                                             ref={buttonRef as ForwardedRef<View>}
                                                             text={translate('walletPage.enableWallet')}
-                                                            onPress={triggerKYCFlow}
+                                                            onPress={() => Navigation.navigate(ROUTES.SETTINGS_ENABLE_PAYMENTS)}
```

Comment:
Shouldn't it need to be `triggerKYCFlow` only? As we are passing `ROUTES.SETTINGS_ENABLE_PAYMENTS` already to `enablePaymentsRoute` of the `KYCWall` component.

Reply:
I've changed it, bc I moved all the steps including add bank account to the ENABLE_PAYMENTS route, but I realized that we might be using enable payments also after adding a debit card (do we? - I couldn't pass the flow bc of card details not being accepted by the BE). I will ask about this flow in the issue